app/models/answer.rb

```diff
@@ -41,5 +45,21 @@ def liked_by(uid)
   def comments
     self.drupal_comments
         .order('timestamp DESC')
+  end
+
+  def answer_notify(current_user)
```

Comment:
This is a lot of code to duplicate. I know I've asked this in other ways, but just one more time to see what you think: is it possible for the Answer model to descend from the Comment model, or some other model, in a class-inheritance where it would inherit these methods instead of having to duplicate them?